pkg/proxy/iptables/proxier.go

```diff
@@ -1321,6 +1324,26 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 	proxier.portsMap = replacementPortsMap
+	// Delete conntrack for all UDP entries
+	proxier.clearUdpConntrackForPort(0)
```

Comment:
If there are any in-flight transactions, this is going to kill them.  Why are you doing this?